sample.cpp

```diff
@@ -110,6 +112,11 @@ class TestSystem : public EntitySystem,
 		std::cout << "An entity was destroyed!" << std::endl;
 	}
 
+	virtual void receive(class World* world, const Events::OnComponentRemoved<Position>& event) override
+	{
+		std::cout << "A component was removed!" << std::endl;
```

Comment:
If you're going to demo `OnComponentRemoved`, then make this line say `A position component was removed!` and add a second event subscriber + implementation for `Rotation`.

Reply:
done